frontend/src/components/auth/PasswordResetSuccess.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Box, Text, Button } from "@chakra-ui/react";
+import { useHistory } from "react-router-dom";
+import { ReactComponent as Logo } from "../../assets/Sistering_Logo.svg";
+
+const PasswordResetSuccessPage = (): React.ReactElement => {
+  const history = useHistory();
+
+  return (
+    <Box display="flex">
+      <Box margin="48px auto" textAlign="center">
+        <Logo />
+        <Text fontSize="xl" fontWeight="bold" marginBottom="24px">
+          Password Change Successful!
+        </Text>{" "}
+        <Text fontSize="l" marginBottom="24px">
+          You may now login with your new credentials.
+        </Text>{" "}
```

Comment:
```suggestion
        <Text fontSize="14px" marginBottom="24px">
          You may now login with your new credentials.
        </Text>
```
Looking at the figma file, I think the text size should be font 14px unless this is inconsistent with our other pages eg: account successfullly created.